components/apimgt/org.wso2.carbon.apimgt.micro.gateway/org.wso2.carbon.apimgt.micro.gateway.status.checker/src/main/java/org/wso2/carbon/apimgt/micro/gateway/status/checker/internal/StatusCheckerServerStartListener.java

```diff
@@ -70,9 +70,8 @@ public void startStatusCheck() {
                                           TimeUnit.MINUTES);
             log.info("StatusCheckerServerStartListener started");
         } else {
-            //Else print an error log
-            log.error("You have not configured the Micro Gateway properly. Please contact cloud@wso2.com, if the " +
-                              "issue persists");
+            //Else print a log indicating that the pinging details are note set
+            log.info("You have not configured the Micro Gateway Ping URL, pinging will not be activated.");
```

Comment:
This configuration is specifically used for a API cloud related task (To periodically send a ping request to a web app deployed in publisher). This is not mandatory configuration for the default micro gateway. IMO, If we set this as a warn, it may highlight the log and user may think this is something he has missed.